test/integration/bank_holidays_test.rb

```diff
@@ -8,18 +8,18 @@ class BankHolidaysTest < ActionDispatch::IntegrationTest
 
     visit "/bank-holidays"
 
-    within 'head' do
-      assert page.has_selector?("title", :text => "UK bank holidays - GOV.UK")
-      desc = page.find("meta[name=description]")
+    within("head", visible: false) do
```

Comment:
This is a bit noisy, I don't suppose there's a block form of this to say "within this block allow inivisible elements" rather than specifying it on each match?  There may not be, just wondering.